src/widgets/vitals/vitals-overview.component.tsx

```diff
@@ -21,13 +20,22 @@ import {
 import { Add24, ChartLineSmooth24, Table24 } from "@carbon/icons-react";
 import dayjs from "dayjs";
 
+import { openWorkspaceTab } from "../shared-utils";
+import {
+  performPatientsVitalsSearch,
+  PatientVitals
+} from "./vitals-card.resource";
+import EmptyState from "../../ui-components/empty-state/empty-state.component";
+import VitalsForm from "./vitals-form.component";
+import styles from "./vitals-overview.scss";
+
 function VitalsOverview() {
+  const config = useConfig();
+  const { t } = useTranslation();
   const initialResultsDisplayed = 3;
-  const [currentVitals, setCurrentVitals] = useState([]);
+  const [currentVitals, setCurrentVitals] = useState<PatientVitals[]>(null);
```

Comment:
Can we use `Array<PatientVitals>` instead of `PatientVitals[]`? This is for consistency and to avoid confusion with tuples `[a, b, ...]`

Reply:
Sure. Fixed.

Reply:
Can we merge this @FlorianRappl?

Reply:
Totally!